internal/command/machine/clone.go

```diff
@@ -44,6 +45,10 @@ func newClone() *cobra.Command {
 			Name:        "name",
 			Description: "Optional name for the new machine",
 		},
+		flag.Bool{
```

Comment:
What should the user do if they want to restore from an older snapshot?

Reply:
Good point :)

We can add another flag `--from-snapshot=ID` and even use `last` as a reference for most recent snapshot. wdyt?

Reply:
Wow, sorry I missed this.  That sounds good to me!

Reply:
Addressed in [this commit](https://github.com/superfly/flyctl/pull/1512/commits/94b98610c024a6a005d7a03b33a848f1ff2bce30#diff-4a3436ee7e8af3a8b58c6dc54d42485e71f9367bc5706d7a6adaaf143e9bcf93R48-R50)